gapii/cc/vulkan_mid_execution.cpp

```diff
@@ -293,7 +293,7 @@ void VulkanSpy::prepareGPUBuffers(CallObserver *observer, PackEncoder *group,
     pool->buffer = nullptr;
 
     memory::Observation observation;
-    observation.set_base(0);
+    observation.set_base(pool->size - 1);
```

Comment:
The point of these observations was to have at least one observations in every pool, so that we can create them correctly on replay side, without having to go through the state tree, find all of the pools, and create them that way.

But since now we have to pre-specify the size of a pool, now these observations occur at the end of the pool, so that the first observations we have encompass the entire pool.

Reply:
Ah okay, it's this weird 0 sized observation thing. Fair enough.

Reply:
A comment here explaining what you just said would be lovely.